webapp/channels/src/components/admin_console/file_upload_setting.tsx

```diff
@@ -1,56 +1,65 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
 import * as Utils from 'utils/utils';
 
 import Setting from './setting';
 
-export default class FileUploadSetting extends Setting {
```

Comment:
why was it extending Setting previously?

Reply:
Probably was a mimic of the other components where we extend from `AdminSetting`.